pkg/config/config_test.go

```diff
@@ -94,7 +124,44 @@ var _ = Describe("Allocation operations", func() {
 		Expect(ipamconfig.LeaderLeaseDuration).To(Equal(3000))
 		Expect(ipamconfig.LeaderRenewDeadline).To(Equal(2000))
 		Expect(ipamconfig.LeaderRetryPeriod).To(Equal(1000))
+	})
+
+	It("overlapping range can be set", func() {
+		var globalConf string = `{
+			"datastore": "kubernetes",
+			"kubernetes": {
+				"kubeconfig": "/etc/cni/net.d/whereabouts.d/whereabouts.kubeconfig"
+			},
+			"log_file": "/tmp/whereabouts.log",
+			"log_level": "debug",
+			"gateway": "192.168.5.5",
+			"enable_overlapping_ranges": false
+		}`
+		Expect(ioutil.WriteFile("/tmp/whereabouts.conf", []byte(globalConf), 0755)).To(Succeed())
```

Comment:
Seems to me we're leaking this file after the execution, but I've noticed that also happens for other tests.

I'll handle that in a follow-up.